src/common/image.c

```diff
@@ -865,8 +865,13 @@ int32_t dt_image_duplicate_with_version(const int32_t imgid, const int32_t newve
     sqlite3_finalize(stmt);
 
     DT_DEBUG_SQLITE3_PREPARE_V2(dt_database_get(darktable.db),
-                                "INSERT INTO main.tagged_images (imgid, tagid)"
-                                "  SELECT ?1, tagid FROM main.tagged_images WHERE imgid = ?2",
+                                "INSERT INTO main.tagged_images (imgid, tagid, position)"
+                                "  SELECT ?1, tagid, "
+                                "        (SELECT (IFNULL(MAX(position),0) & (4294967295 << 32))"
```

Comment:
4294967295 << 32  ???? Isn't this expected to be "2 << 32" ?

The above is a 64 bits integer where the 32 first bits (high order) are set to 1. Maybe expected, but I prefer to ask, not clear reading the code.